RecoLocalFastTime/FTLRecProducers/python/mtdUncalibratedRecHits_cfi.py

```diff
@@ -12,8 +12,8 @@
     timeCorr_p0 = cms.double( 2.21103),
     timeCorr_p1 = cms.double(-0.933552),
     timeCorr_p2 = cms.double( 0.),
-    c_LYSO = cms.double(138.46235),
-    bar_length = cms.double(57.0)
+    positionError = cms.double( 0.6),  #in unit cm
```

Comment:
is this supposed to be the same 0.6 as in ```BTLRecHitsErrorEstimatorIM```?
If this is not some numeric coincidence, the parameters should be explicitly the same.
Perhaps the more practical way is to extract this 0.6 from a call to ```BTLRecHitsErrorEstimatorIM```

Reply:
Hi, 
Just to make sure, to make the error 0.6 to be able to call from the `BTLRecHitsErrorEstimatorIM`, I guess I can make the `positionError` as a private member in that class and add a new function that returns the value of `positionError` so I can just call the function to get the position error. Also, the position error could be set in the constructor if it's needed. Would that sound reasonable?

Thanks.

Reply:
> Would that sound reasonable?

It sounds reasonable to have a separate member method for this.

Reply:
Hi,
So do you mean that I don't need to set a private member for it but just make a member method that returns 0.6? But I feel that maybe having that in private will be more elegant so we can change it in the future if necessary.
Thanks.

Reply:
the point is how to make this 0.6 in .py to be instead available for BTLUncalibRecHitAlgo via ```BTLRecHitsErrorEstimatorIM``` (or another common source). 
There are multiple places where instances of ```BTLRecHitsErrorEstimatorIM``` are made (currently, already MTDCPEFromSiPMTimeBTL and MTDThresholdClusterizer; add to this my proposal to also use it in BTLUncalibRecHitAlgo so that the same 0.6 can be consistently available everywhere.
This means that this 0.6 can not be a constructor argument, unless the ``BTLRecHitsErrorEstimatorIM``` is served from a single place that can construct it, which knows of (possibly configurable) 0.6 .

Reply:
Hi,
I just realized that `BTLRecHitsErrorEstimatorIM` should be initialized with `MTDGeomDet` and `LocalPoint`, which are not available in `MTDUncalibratedRecHitsProducer`. Do you have any suggestions to do that?
Thanks.

Reply:
if 0.6 is hardcoded, it can be returned with a static method, no need to actually instantiate.